### DISCUSSION TITLE
How to setup quick demo to show a client

### DISCUSSION THREAD

**Discussion by toms3t** [saleor/saleor#13385] [Q&A]
Hi team -
I'm trying to setup a quick demo for a client to show what the storefront would look like with a couple of their products. I've got the local environment set up but I'm not able to figure out how to create a product in the admin dashboard and have it reflected in the storefront. Can anyone tell me what I'm doing wrong?

**Comment by timuric** [comment]
Hi @toms3t did you enable the product in respective channel?

**Reply by toms3t** [reply]
@timuric What's the process to enable the product in the channel? I'm using "default-channel" if that matters.

**Reply by timuric** [reply]
@toms3t you can use graphql inspector to learn how the dashboard communicates with catalog https://chrome.google.com/webstore/detail/graphql-network-inspector/ndlbedplllcgconngcnfmkadhokfaaln?hl=en-GB

**Reply by bhawani0x** [reply]
hi @timuric is there any video or document for local setup, every time I try I face issue
